scripts/output_env.sh

```diff
@@ -22,11 +22,30 @@ echo "* Operating system and architecture:"
 uname -a
 
 echo
-if `hash armcc > /dev/null 2>&1`; then
-    echo "* armcc:"
-    armcc --vsn | head -n 2
+if [ -n "${ARMC5_CC+set}" ]; then
+    if `hash "$ARMC5_CC" > /dev/null 2>&1`; then
+        echo "* $ARMC5_CC at environment variable 'ARMC5_CC':"
+        $ARMC5_CC --vsn | head -n 2
+    else
+        echo "* $ARMC5_CC at environment variable 'ARMC5_CC' not found!"
+    fi
 else
-    echo "* armcc not found!"
+    if `hash armcc > /dev/null 2>&1`; then
+        echo "* armcc:"
+        armcc --vsn | head -n 2
+    else
+        echo "* armcc not found!"
+    fi
+fi
+
+if [ -n "${ARMC6_CC+set}" ]; then
+    echo
+    if `hash "$ARMC6_CC" > /dev/null 2>&1`; then
+        echo "* $ARMC6_CC at environment variable 'ARMC6_CC':"
+        $ARMC6_CC --vsn | head -n 2
+    else
+        echo "* $ARMC6_CC at environment variable 'ARMC6_CC' not found!"
```

Comment:
I can do `hash armclang` if your prefer. However, I do not think it is sensible to have both `armcc` and `armclang` in the path simultaneously. This is because these toolchains have executables with names that overlap. For instance, `armar` and `armlink` are called the same in both cases. If you have both bin folders in the path, then you will potentially be compiling with `armclang` or `armcc` as desired BUT you might be using the wrong version of other tools such as `armar`. I tried this and it definitely fails.